src/components/ExpandableSection/__tests__/ExpandableSection.spec.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { Button } from '../../../';
+import ExpandableSection from '../ExpandableSection';
+
+const getExpandableSection = (props = {}) => shallow(<ExpandableSection id="51er" {...props} />);
+
+
+describe('<ExpandableSection />', () => {
+  it('renders right class name', () => {
+    const mounted = getExpandableSection();
+    mounted.setProps({ open: true });
+    expect(mounted.hasClass('slds-section')).toBeTruthy();
+  });
+
+  it('renders children', () => {
+    const child = <div className="51ers" />;
+    const mounted = getExpandableSection({ id: 'test', children: child });
+    mounted.setProps({ open: true });
+    expect(mounted.contains(child)).toBeTruthy();
+  });
+
+  it('renders a title', () => {
+    const mounted = getExpandableSection();
+    mounted.setProps({ title: 'Title', open: false });
```

Comment:
`const mounted = getExpandableSection({ title: 'Title', open: false });` no need to use setProps here if you use a fn to get the component